.jenkins/caffe2/test.sh

```diff
@@ -75,9 +73,9 @@ for test in $INSTALL_PREFIX/test/*; do
   esac
 
   "$test" --gtest_output=xml:"$TEST_DIR"/cpp/$(basename "$test").xml
-  tmp_exit_code="$?"
-  if [ "$exit_code" -eq 0 ]; then
-    exit_code="$tmp_exit_code"
+  exit_code="$?"
```

Comment:
you don't need to capture the exit_code anymore. The whole purpose of `set -e` is to let it automatically exit if exit code of last command is non-zero.